functorch/_src/aot_autograd.py

```diff
@@ -886,6 +899,37 @@ def aot_module_simplified(mod: nn.Module, *top_args, **top_kwargs) -> nn.Module:
     params_flat = tuple(params_flat)
     params_len = len(params_flat)
 
+    # TODO(voz): Pull up to dynamo
+    # See [Real vs Fake Parms] below
+    fake_mode = None
+
+    def fakify_params_and_buffers(flat_args):
```

Comment:
You sort of cover this in the Note below, but (1) honestly the note should get pulled up earlier and (2) a direct discussion about why inputs is always fake, but not always params/buffers would be helpful for readers